app.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+"use strict";
+
+{
+  let fizz = 0;
+  let buzz = 0;
+  let fizzbuzz = 0;
+  for (let i = 0; i < 99; i++) {
+    fizz++;
+    buzz++;
+    fizzbuzz++;
+    if (fizz % 15 === 0) {
```

Comment:
わざわざ変数を用意しなくても、以下で実現できると思います。

```suggestion
    if (i % 15 === 0) {
```

Reply:
ありがとうございます。
変更してみます！